_posts/2023-06-30-july-10-downtime.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Downtime July 10th, 2023
+state: active
+severity: 0
+---
+
+We will be performing upgrades to some of our hardware. Much of our
```

Comment:
We should add here that this will potentially last until end of the day on the 11th.